include/module_hello.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2021 Legrand North America, LLC.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/** @file
+ *  @brief Module Hello API
+ */
+#ifndef EXAMPLE_APPLICATION_INCLUDE_MODULE_HELLO_H_
+#define EXAMPLE_APPLICATION_INCLUDE_MODULE_HELLO_H_
```

Comment:
Should this subsystem API file be in a subsystem-specific folder (i.e., include/module_hello/module_hello.h, giving #include <module_hello/module_hello.h>)?

Reply:
Unless there is a compelling reason they should not, example code should follow the pattern(s) in Zephyr = yes.